src/components/InputsTable.js

```diff
@@ -140,15 +93,9 @@ const InputsTable = props => {
   return h(AutoSizer, [({ width, height }) => {
     return h(div, {}, [
       structBuilderVisible && h(StructBuilderModal, {
-        structBuilderName: _.get(buildStructNamePath(structBuilderPath), inputTableData),
-        structBuilderBreadcrumbs: buildStructBuilderBreadcrumbs(structBuilderPath, inputTableData),
-        structBuilderInputType: _.get(buildStructInputTypePath(structBuilderPath), inputTableData),
+        inputTableData,
```

Comment:
Nice! This feels a lot neater to me!

Happy to approve this now, with one final (non-critical, TOL-level) thought: `inputTableData` feels very "grab bag of stuff"-y to me. Are you using all of it, or it is only a specific subset you really need? (eg, could you replace this passing of the entire `inputTableData` object with only the fields that the struct builder really needs (maybe just the set of struct paths for this particular struct?)

Reply:
The funny thing is: this exact thought is what caused me to put all of those `buildStructPath` functions in the InputTable component in the first place. The purpose of those functions is to pull out the parts of `inputTableData` that the struct builder needs.

But actually, I think there may be an in-between that will achieve what you're suggesting *and* make the `buildStructPath` functions slightly less complicated. I'll give it a try in the time between now and sprint planning...